prov/efa/src/efa_prov_info.c

```diff
@@ -73,7 +73,7 @@ const struct fi_domain_attr efa_domain_attr = {
 	.control_progress	= FI_PROGRESS_AUTO,
 	.data_progress		= FI_PROGRESS_AUTO,
 	.resource_mgmt		= FI_RM_DISABLED,
-	.mr_mode		= OFI_MR_BASIC_MAP | FI_MR_LOCAL | FI_MR_BASIC,
+	.mr_mode		= OFI_MR_BASIC_MAP | FI_MR_LOCAL | OFI_MR_BASIC,
```

Comment:
I believe efa code has more occurrence for `FI_LOCAL_MR` to handle old API, but it's ok to leave as is for this PR

Reply:
There is only one occurrence in efa unit test, which I think should leave it as it is (tests are only supposed to see public headers).

Reply:
yep yep